client/components/marked-lines/index.jsx

```diff
@@ -88,7 +88,7 @@ const MarkedLines = ( { context } ) => {
 
 					return (
 						<div
-							key={ content }
+							key={ lineNumber }
```

Comment:
Why did you decide to use `line-${ lineNumber }` in one case and `${lineNumber}` in the other? I think any of those would work the same, so we could use the same key in both cases (to avoid confusions).

Reply:
I didn't realize that they would both work the same. Will fix it soon. 
Thanks :)

Reply:
To be more precise, I don't think it makes a difference because both collections have different parents and keys don't have to be unique across the app, they only need to be unique among children of a parent.